addons/website_sale_slides/views/website_sale_templates.xml

```diff
@@ -30,7 +30,8 @@
                            <t t-if="course.total_time"> - </t><t t-out="course.total_slides"/> step(s)
                         </t>
                     </div>
-                    <a role="button" class="btn btn-primary ms-2 my-2" t-attf-href="/slides/#{slug(course)}">
+                    <t t-set="signup_token" t-value="order.partner_id.signup_prepare() and order.partner_id.signup_token"/>
+                    <a role="button" class="btn btn-primary ms-2 my-2" t-attf-href="/web/login?token=#{signup_token}&amp;redirect=/slides/#{slug(course)}">
```

Comment:
Actually this link is also a few lines above (the course name is a link), it should probably be adapted as well?